main/mail/stubs/dovecot.conf.mas

```diff
@@ -124,6 +124,7 @@ mail_gid=<% $gid %>
 
 namespace inbox {
     inbox=yes
+    separator = /
```

Comment:
You cannot change the IMAP separator without migrating code as the current folders will disappear. My proposal here is to change the configuration to use `.` as separator. See the untested hints below.